### PR TITLE
Refactor and fix simplified-ui-accounts, simplified-ui-settings and simplified-ui-profiles

### PR DESCRIPTION
**Pull request by qnga**
**What's this do?**
- Introduce ViewModels where there were missing.
- Post background events to the UI thread in ViewModels.
- Don't use `runOnUIThread` any more in Fragments to ensure there's no crash on configuration changes.
- Indicate whether the build supports AxisNow in the debug settings.

**Why are we doing this? (w/ JIRA link if applicable)**
To ensure that configuration changes don't cause a crash.

**How should this be tested? / Do these changes have associated tests?**
This can be tested trying every screen from the relevant modules.

**Dependencies for merging? Releasing to production?**
N/A

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
Yes, I did.

### CODE REVIEW

File: simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountFragment.kt [hunk 1]
```diff
@@ -894,29 +841,22 @@ class AccountFragment : Fragment() {
           view,
           object : com.squareup.picasso.Callback {
             override fun onSuccess() {
-              this@AccountFragment.uiThread.runOnUIThread {
-                onSuccess.invoke()
-              }
+              onSuccess.invoke()
```
Comment (io7m) [inline simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountFragment.kt:844]: Are the Picasso callbacks guaranteed to be called on the UI thread? It's been a very long time, I'm unsure what guarantees they provide.

File: simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListRegistryViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  }
+
+  private fun onAccountEvent(event: AccountEvent) {
+    this.accountCreationEvents.onNext(event)
+  }
+
+  override fun onCleared() {
+    super.onCleared()
+    this.subscriptions.clear()
+    this.backgroundExecutor.shutdown()
```
Comment (io7m) [inline simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListRegistryViewModel.kt:60]: :+1: 

So many people forget to shut down exectors. They extend `AutoCloseable` in modern Java, but ... you know... Android...

File: simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountFragment.kt (hunk 1 above)
Comment (qnga) [reply simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountFragment.kt:844]: I couldn't find any guarantee in the doc. Looking at the code, it is so though. Everything happens on the same thread on completion: the view image is set and then the callback is called. I think it's unlikely to change in the future.

File: simplified-ui-settings/src/main/java/org/nypl/simplified/ui/settings/SettingsFragmentDebug.kt [hunk 3]
```diff
@@ -166,7 +86,33 @@ class SettingsFragmentDebug : Fragment() {
...
       view.findViewById(R.id.settingsVersionCrashlyticsID)
 
-    return view
+    this.drmTable.addView(
+      this.createDrmSupportRow("Adobe Acs", this.viewModel.adeptSupported)
+    )
+    this.drmTable.addView(
+      this.createDrmSupportRow("AxisNow", this.viewModel.axisNowSupported)
+    )
+
```
Comment (io7m) [inline simplified-ui-settings/src/main/java/org/nypl/simplified/ui/settings/SettingsFragmentDebug.kt:95]: Nice to see this! This has been something I've been meaning to do for a while. We have quite a few optional DRM components that could be added here, but they don't need to go in right now.

File: simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountFragment.kt (hunk 1 above)
Comment (io7m) [reply simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountFragment.kt:844]: Yeah, I doubt they would change this.

**Comment by io7m** [comment]
I like it!

**Review by io7m — APPROVED** [review]

**Comment by qnga** [comment]
Thanks!

**Pull request merged**